File: aparcamiento/AMarotoChivite-aparcamiento/AMarotoChivite-aparcamiento-Java/src/test/java/simulacionParking/models/VigilanteTest.java
package simulacionParking.models;

import org.junit.jupiter.api.Test;
import simulacionParking.factories.ParkingFactory;
import simulacionParking.factories.VehiculoFactory;

import static org.junit.jupiter.api.Assertions.assertTrue;

class VigilanteTest {

    @Test
    void parkingCompletoTest() {
        Parking parking = ParkingFactory.create();

        for (int filas = 0; filas < parking.matriz().length; filas++) {
            for (int columnas = 0; columnas < parking.matriz()[filas].length; columnas++) {
                parking.matriz()[filas][columnas] = VehiculoFactory.create();
            }
        }

        assertTrue(Vigilante.parkingCompleto(parking));
    }
}

File: aparcamiento/AMarotoChivite-aparcamiento/AMarotoChivite-aparcamiento-Kotlin/src/test/kotlin/simulacionParking/models/VigilanteTest.kt
package simulacionParking.models

import org.junit.jupiter.api.Test
import simulacionParking.factories.ParkingFactory
import simulacionParking.factories.VehiculoFactory
import kotlin.test.assertTrue

internal class VigilanteTest {
    @Test
    fun parkingCompletoTest() {
        val parking = ParkingFactory.create()
        for (filas in 0 until parking.matriz.size) {
            for (columnas in 0 until parking.matriz[filas].size) {
                parking.matriz[filas][columnas] = VehiculoFactory.create()
            }
        }

        assertTrue(Vigilante.parkingCompleto(parking))
    }
}